src/utils.ts

```diff
@@ -11,7 +11,6 @@ export function StripHtml(html: string) {
     return `${process.env.PUBLIC_URL}/images/portraits/${dotaName}_png.png`;
  }
 
- // TODO capitalize
  export const SUPPORTED_LANGUAGES = {

```

Comment:
Is there a reason why one wouldn't just modify this object directly?

Also this object seems backwards.  Short codes as the value and the human string as the key?

🤔

Reply:
This might require additional work to refactor where the language is set in redux, but it seems like a better design?

Reply:
Your concerns on the object's structure, and future use are for sure valid and I invite the conversation to go over the issue outlined and scope of refactoring.
As for why not just modify the object directly, because it is currently being touched by a few other pieces in the code base (components and material-ui frame work all expecting "language": "lang-code"). I figured the swift direct approach to fix "language capitalization" (for display purpose) was to create a helper function to handle formatting for displaying purposes.

Reply:
I agree it seems backwards and we could probably flip it. @albertcui do you remember if this was done for a reason? Are we indexing into this object by the human name?